lib/api/views.js

```diff
@@ -1,16 +1,45 @@
-var linz = require('../../'),
-    path = require('path');
+'use strict';
+
+const linz = require('../../');
+const path = require('path');
+const utils = require('../utils');
 
 /**
  * Return the path to Linz's view directory, appending the specific view if passed in.
- * @param {String} view  The view to append to the view directory path (a Linz specific view).
- * @return {String}      A path to a particular view file.
+ * @param {String} view The view to append to the view directory path (a Linz specific view).
+ * @return {String}     A path to a particular view file.
  * @api public
  */
-function viewPath (view) {
-    return path.resolve(__dirname, '..', '..', 'views', (view || ''));
-}
+const viewPath = view => path.resolve(__dirname, '..', '..', 'views', (view || ''));
+
+/**
+ * Render a template in the context of Linz.
+ * @param  {Object}   data     Data and options to pass to the template.
+ * @param  {Function} callback Optional callback function.
+ * @return {String}            Renders and returns the template HTML.
+ */
+const render = (data, callback) => {
+
+    const appLocals = {
+        linzNavigation: linz.get('navigation'),
+        template: 'wrapper'
+    };
+
+    const locals = utils.merge(appLocals, data || {});
```

Comment:
@allanchau, try this... `const locals = Object.assign(appLocals, clone(data || {}));` (you'll need to require `clone` of course).